perl/p5-data-validate-domain/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+perl5.branches      5.26 5.28 5.30
+perl5.setup         Data-Validate-Domain 0.14
+
+platforms           darwin
+maintainers         nomaintainer
+license             Artistic-1 GPL
```

Comment:
This is normally:

`license             {Artistic-1 GPL}
`

Reply:
Thanks done.

Reply:
?

Reply:
Oops thx.